tools/azure-sdk-tools/devtools_testutils/sanitizers.py

```diff
@@ -604,15 +611,14 @@ def _send_sanitizer_request(sanitizer: str, parameters: dict) -> None:
     if is_live_and_not_recording():
         return
 
-    response = requests.post(
+    http_client.request(method="POST", url=
         "{}/Admin/AddSanitizer".format(PROXY_URL),
```

Comment:
Looks like the URL parameter might have been disrupted by a batch update?
```suggestion
    http_client.request(method="POST",
        url="{}/Admin/AddSanitizer".format(PROXY_URL),
```

Reply:
This looks like a spacing update to me. I'll make certain to apply `black`